### PR TITLE
docs: add docs about pouch cri introduction

### PR DESCRIPTION
**Pull request by liyanyanli**
Signed-off-by: yanli <yanliyanli1993@gmail.com>

1.Describe what this PR did
This PR adds pouch cri introduction

2.Does this pull request fix one issue?
 #486

3.Describe how you did it
NONE

4.Describe how to verify it
NONE

5.Special notes for reviews
NONE

### CODE REVIEW

File: docs/pouch_with_cri_introduction.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# pouch with CRI introduction
```
Comment (Letty5411) [inline docs/pouch_with_cri_introduction.md]: Please use capital letter for the first word `Pouch`

File: docs/pouch_with_cri_introduction.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## what is CRI
```
Comment (Letty5411) [inline docs/pouch_with_cri_introduction.md]: what -> What

File: docs/pouch_with_cri_introduction.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The Container Runtime Interface (CRI) is a plugin interface which enables kubelet to use a wide variety of container runtimes, without the need to recompile. 
+CRI (_Container Runtime Interface_) consists of a [protobuf API](https://git.k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto),specifications/requirements and [libraries](https://git.k8s.io/kubernetes/pkg/kubelet/server/streaming)for container runtimes to integrate with kubelet on a node.
+
+This diagram explains how the Container Runtime Interface works:
+
+![CRI works](static_files/pouch_cri_works.png)
+
+## why need to implement CRI in pouch
+
+The purpose of the pouch CRI implementation is to integrate with Kublete.This allow Kubernetes to directly launch and manage containers through the pouch CRI.
```
Comment (Letty5411) [inline docs/pouch_with_cri_introduction.md]: allow -> allows

File: docs/pouch_with_cri_introduction.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## what is CRI
+
+The Container Runtime Interface (CRI) is a plugin interface which enables kubelet to use a wide variety of container runtimes, without the need to recompile. 
+CRI (_Container Runtime Interface_) consists of a [protobuf API](https://git.k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto),specifications/requirements and [libraries](https://git.k8s.io/kubernetes/pkg/kubelet/server/streaming)for container runtimes to integrate with kubelet on a node.
+
+This diagram explains how the Container Runtime Interface works:
+
+![CRI works](static_files/pouch_cri_works.png)
+
+## why need to implement CRI in pouch
```
Comment (Letty5411) [inline docs/pouch_with_cri_introduction.md]: why-> Why
 Same comments for the following titles.

File: docs/pouch_with_cri_introduction.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+## application scene
+
+Manage containers via Kubernetes.
+
+## pouch CRI Create a single-container pod
+
+Let's use an example to demonstrate how pouch-cri works for the case when Kubelet creates a single-container pod:
+
+* Kubelet calls pouch-cri, via the CRI runtime service API, to create a pod.
+* pouch uses containerd to create and start a special pause container (the sandbox container) and put that container inside the pod’s cgroups and namespace (steps omitted for brevity).
```
Comment (Letty5411) [inline docs/pouch_with_cri_introduction.md]: pause -> paused?

File: docs/pouch_with_cri_introduction.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* pouch configures the pod’s network namespace using CNI.
+* Kubelet subsequently calls pouch-cri, via the CRI image service API, to pull the application container image.
+* Kubelet then calls pouch-cri, via the CRI runtime service API, to create and start the application container inside the pod using the pulled container image.
+* pouch finally calls containerd to create the application container, put it inside the pod’s cgroups and namespace, then to start the pod’s new application container.
+
+After these steps, a pod and its corresponding application container is created and running.
```
Comment (Letty5411) [inline docs/pouch_with_cri_introduction.md]: is -> are

File: docs/pouch_with_cri_introduction.md (hunk 5 above)
Comment (liyanyanli) [reply docs/pouch_with_cri_introduction.md]: In Kubernetes, the pause container serves as the "parent container" for all of the containers in pod.

File: docs/pouch_with_cri_introduction.md (hunk 6 above)
Comment (liyanyanli) [reply docs/pouch_with_cri_introduction.md]: it is a single container

**Comment by Letty5411** [comment]
LGTM

**Pull request merged**